dags/tasks/sinopia/rdf2marc.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import json
+from urllib.parse import urlparse
+from os import path, getenv
+
+from airflow.contrib.hooks.aws_lambda_hook import AwsLambdaHook
+
+
+def Rdf2Marc(**kwargs):
+
+    task_instance = kwargs["task_instance"]
+    instance_uri = task_instance.xcom_pull(task_ids="sqs-sensor")
+
+    """Runs rdf2marc on a BF Instance URL"""
+    instance_path = urlparse(instance_uri).path
+    instance_id = path.split(instance_path)[-1]
+
+    rdf2marc_lambda = getenv("RDF2MARC_LAMBDA")
```

Comment:
Since we are using a single environment for Airflow, I think the `RDF2MARC_LAMBDA` and `MARC_S3_BUCKET` should be set per Sinopia environment (development, stage, and production) as Airflow variables, i.e. ``RDF2MARC_LAMBDA_DEV`, `RDF2MARC_LAMBDA_STAGE` etc.